USE-CASES/W3C-LBD-CG.md

```diff
@@ -196,16 +88,14 @@ Another related case in smart buildings, which would greatly benefit from harmon
             "osh": "https://w3id.org/ibp/osh/OpenSmartHomeDataSet#",
             "bot": "https://w3id.org/bot#",
             "sosa": "http://www.w3.org/ns/sosa/",
```

Comment:
should add `"brick": "https://brickschema.org/schema/1.1/Brick#",` to `@context`